### DISCUSSION TITLE
Setting the % of Zoom with Scroll Wheel

### DISCUSSION THREAD

**Discussion by bnpwalter** [freeplane/freeplane#811] [Help]
I've been unable to find the means to control the % of Zooming In/Out using the mouse scroll wheel.  Currently, Zooming with my mouse is set too high.  I haven't been able to find in preferences or literature how to control the amount of Zoom when scrolling.

**Comment by dpolivaev** [comment]
Currently, there is no such option. Actually, it was not needed until Windows introduced an option to change Mouse Scroll Speed for all apps. Crazy option, because it would make sense only on a per-app basis. So I am going to add this configuration option in the coming Freeplane versions.

**Reply by Lotus49b** [reply]
Hi Dimitry

Just following up on this. 

It would be great to have the ability for more granular zoom levels. 

I find that, on my display, 50% is too small to read but at the next level (65%) the map is too large and I lose the "bigger picture".

Thanks
